core/proto/signature/gen.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
We call this file `doc.go` in other places. Should we keep the same naming consistent? I'm fine with either.